website/docs/cloud-docs/overview/activate-payg.mdx

```diff
@@ -13,7 +13,7 @@ Pay-as-you-go offers dynamic billing based on monthly consumption. Larger organi
 
 ## Activate pay-as-you-go
 
-To enable pay-as-you-go and change your organization's plan, navigate to **Organization Settings > Plan & Billing**. 
+To enable pay-as-you-go and change your organization's plan, log in to [HCP Terraform](https://app.terraform.io), then navigate to **Organization Settings > Plan & Billing**. 
```

Comment:
Can you use "sign in" instead of "log in"? It's what we use in the platform itself and what I'm using in my [HCP Link plan](https://docs.google.com/document/d/1sl6qPEAZZPTzv8NsSaEpFe_YnikTDnNbDo25RqTqaaU/edit?usp=sharing)

Reply:
Oh, thanks for sharing this! Updated!